crates/vizia_winit/src/application.rs

```diff
@@ -30,9 +36,29 @@ use winit::{
     event_loop::{ControlFlow, EventLoop, EventLoopProxy},
 };
 
+#[derive(Debug)]
+pub enum UserEvent {
+    Event(Event),
+    #[cfg(not(target_arch = "wasm32"))]
+    AccessKitActionRequest(accesskit_winit::ActionRequestEvent),
```

Comment:
If this is a winit limitation, can we avoid changing the public interfaces and avoid the into() calls?

Reply:
This is because accesskit is utilising winit custom events to receive action requests from a screen reader. But we're already using custom events to send vizia events between threads with the event loop proxy. So this enum is required so we can keep both functionality. I'm open to suggestions on API improvements if something can be done.